plugins/screen-login/manifest/ios.json

```diff
@@ -0,0 +1,34 @@
+{
+  "api": {},
+  "dependency_repository_url": [],
+  "platform": "ios",
+  "dependency_name": "@applicaster/screen-login",
```

Comment:
This is only for mobile, so it doesn't matter, but I'm telling you anyway

on the web based platforms (samsung TV, LG TV), we use webpack for bundling the app, instead of the metro bundler. The consequence is that node_modules are not transformed by default.

We didn't want to transform all node_modules as it has a performance cost in development, so we added a rule to transform all plugins which are prefixed with `zapp-react-` or `quick-brick`
so Ideally your package name would be `@applicaster/quick-brick-screen-login`

Reply:
got it 👍